chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
OK, I understand it. It is OK to set the initial value of `previous_epoch_detail` to `-1`. In that case, we should explicitly write this specification in the document. Personally, I prefer it as it makes the implementation simpler.